sources/platform/actors/development/actor_definition/input_schema/specification.md

```diff
@@ -183,16 +183,18 @@ Rendered input:
+| `enum`          | [String]                                                                                                                                                                                                              | Required if <br/>`editor` <br/>is `select` | Using this field, you can limit values <br/>to the given array of strings. <br/>Input will be displayed as select box.                                                                                                                                                                         |
+| `enumTitles`    | [String]                                                                                                                                                                                                              | No                                         | Titles for the `enum` keys described.                                                                                                                                                                                                                                                          |
+| `nullable`      | Boolean                                                                                                                                                                                                               | No                                         | Specifies whether `null` <br/>is an allowed value.                                                                                                                                                                                                                                             |
+| `isSecret`      | Boolean                                                                                                                                                                                                               | No                                         | Specifies whether the input field<br />will be stored encrypted.<br />Only available <br />with `textfield` and `textarea` editors.                                                                                                                                                            |
+| `allowAbsolute` | Boolean                                                                                                                                                                                                               | No                                         | Used only with `datepicker` editor for absolute date value picker in form `YYYY-MM-DD`. This defaults to `true`.                                                                                                                                                                               |
```

Comment:
The properties are fine, but I don't understand what they mean well. Perhaps there should be better explanation. Is it:
- If I enable allow absolute then I am able to pick certain date
- If I enable allow relative then I can type "3 months"??
But what is the returned value is what? Always a date in ISO format?

Reply:
I think the `allowRelative` explanation is good, but the `allowAbsolute` can be improved. Also could maybe show an example like we do it for other more complicated fields.

Reply:
So, in this case, we should have 2 different editors or "resources types". You should not have varying outputs from a single component.

Reply:
Like one function should always return either date object or relative string but not both.

Reply:
But the main requirement from store team is to have a single field

Reply:
Ok, that makes sense then.